Basic-Car-Maintenance/Shared/Utilities/Constants.swift

```diff
@@ -16,3 +16,34 @@ enum FirestoreCollection {
 enum FirestoreField {
     static let userID = "userID"
 }
+
+enum SFSymbol {
+    //Tabview icons
```

Comment:
```suggestion
    // TabView
```
Fix all the comments, add a space after the `//`
And remove the word "icons" they're all icons cause this is an enum of SF symbols, so it's redundant